gemrb/core/Scriptable/Actor.cpp

```diff
@@ -7280,7 +7280,7 @@ void Actor::UpdateActorState()
 		//IN BG1 and BG2, this is at the ninth frame... (depends on the combat bitmap, which we don't handle yet)
 		// however some critters don't have that long animations (eg. squirrel 0xC400)
 		if ((frameCount > 8 && currentFrame == 8) || (frameCount <= 8 && currentFrame == frameCount/2)) {
-			GetCurrentArea()->AddProjectile(attackProjectile, Pos, LastTarget, false);
+			GetCurrentArea()->AddProjectile(attackProjectile);
```

Comment:
Was the intention to launch it even if the creature got obliterated in the meanwhile or what? Wondering if LastTargetPersistent would work instead for whatever you want to achieve with this commit.

Reply:
I have attack hits without damage before edits sometime.

Reply:
You mean visual hits or something else? Because the games animate fake attacks to make it seem more is going on. I think we don't implement that part though, it's basically #92.

What weapons and circumstances were involved?

Reply:
This can happen if ClearActions was called (which resets LastTarget) between the creation of the Attack projectile and the call to GetCurrentArea()->AddProjectile.

Reply:
ok, LastTargetPersistent is immune to that.

Reply:
Done refix it